### ISSUE TITLE
Crashing with `attempt to get length of field 'playerForces' (a nil value)`

### ISSUE THREAD

**Issue by NeveHanter** [veden/Rampant#76]
I will be cross-posting that bug report as I'm not sure if it's really caused by Krastorio 2 or the Rampant. It doesn't occur when I start a fresh game.

```
The mod Krastorio 2 (1.2.16) caused a non-recoverable error.
Please report this error to the mod author.

Error while running event Krastorio2::on_configuration_changed
The mod Rampant (2.2.0) caused a non-recoverable error.
Please report this error to the mod author.

Error while running event Rampant::on_force_created (ID 28)
__Rampant__/control.lua:845: attempt to get length of field 'playerForces' (a nil value)
stack traceback:
	__Rampant__/control.lua:845: in function <__Rampant__/control.lua:844>
stack traceback:
	[C]: in function 'create_force'
	__Krastorio2__/scripts/util.lua:5: in function 'ensure_turret_force'
	__Krastorio2__/scripts/migrations.lua:24: in function 'generic'
	__Krastorio2__/control.lua:60: in function <__Krastorio2__/control.lua:58>
```

### Reproduction
- Load the attached save game,
- Observe the error

Factorio version: 1.1.52
Player name: NeveHanter

Save game: https://drive.google.com/file/d/1CwG_St0FSnHGF3ed6FTHCNo4NsNJGlVB/view?usp=sharing

**Comment by NeveHanter** [comment]
Krastorio 2 GitHub issue: https://github.com/raiguard/krastorio-2/issues/164

**Comment by veden** [comment]
this should be fixed in the version on the master branch

**Comment by veden** [comment]
this should be fixed in 3.0.0
